### PR TITLE
DPC-1737 ignore snyk finding

### PR DESCRIPTION
**Pull request by MrBilnon**
### Fixes [DPC-1737](https://jira.cms.gov/browse/DPC-1737)

Snyk complains about one our libraries (mock-server) subject to CORs vulnerabilities.  This library is only used in our tests and does not run in production.  

### Change Details

* ignore

### Security Implications

- [x] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [x] security checklist is completed for this change
https://confluence.cms.gov/display/DAPC/2021-09-21%3A+DPC+4.3+Security+Checklist

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### CODE REVIEW

**Review by dhgreene — DISMISSED** [review]
👍

File: .snyk [hunk 1]
```diff
@@ -133,4 +133,9 @@ ignore:
         reason: not using the vulnerable version
         expires: 2022-10-15T15:32:25.097Z
         created: 2021-09-15T15:32:25.107Z
+  SNYK-JAVA-ORGMOCKSERVER-1566476:
+    - '*':
+        reason: this is test code, signed off in DPC-1737
+        expires: 2021-10-29T14:56:29.774Z
```
Comment (dhgreene) [inline .snyk:139]: Should the expiration be a bit longer? Do we need an expiration for this particular issue?

Comment (MrBilnon) [reply .snyk:139]: @dhgreene expiration is required, I can make it one year out

Comment (dhgreene) [reply .snyk]: I'm reasonably certain the `expires` line can be omitted.

Comment (MrBilnon) [reply .snyk]: ok

**Review by dhgreene — APPROVED** [review]

**Pull request merged**